UnrealPlugin/License.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
the repo itself is using Apache license, can we have that here and in the code files as well? [https://github.com/PlayFab/gsdk/blob/master/LICENSE](https://github.com/PlayFab/gsdk/blob/master/LICENSE)